tsconfig.json

```diff
@@ -18,9 +18,13 @@
     "typeRoots": [
       "node_modules/@types",
       "node_modules/@typed-mxgraph"
-    ]
+    ],
+    "stripInternal": true
   },
   "include": [
     "src/**/*"
+  ],
+  "exclude": [
+    "src/api/public/api.ts"
```

Comment:
I think that it is a problem. When I run `npm run docs`, I have the error: ```Warning: Unable to locate entry point: /****/BPMN-Visualization/src/api/public/api.ts``` and the generated doc is empty.

Reply:
yes indeed the problem was here